software/bcftools/stats/main.nf

```diff
@@ -21,8 +21,8 @@ process BCFTOOLS_STATS {
     tuple val(meta), path(vcf)
 
     output:
-    tuple val(meta), path("*.txt"), emit: stats
-    path  "*.version.txt"         , emit: version
+    tuple val(meta), path("*stats.txt"), emit: stats
+    path  "*.version.txt"              , emit: version
```

Comment:
Change the meta.yml file accordingly

Reply:
Not sure I understand?

Reply:
https://github.com/nf-core/modules/pull/164/commits/d2508e3c0308ff601aa30b5f21d3a2e1eac85d75

Reply:
My mistake, my comment was wrong.